scripts/apps/vocabularies/controllers/VocabularyConfigController.ts

```diff
@@ -11,11 +11,13 @@ function getOther() {
 }
 
 export interface IScope extends IDirectiveScope<void> {
+    filterVocab(vocabulary: IVocabulary): boolean;
     vocabularies: Array<IVocabulary>;
     vocabulary: any;
     tags: IVocabulary['tags'];
     loading: boolean;
     mediaTypes: object;
+    localQuery: string;
```

Comment:
`localQuery` was a bad suggestion, since it's no longer limited to a template but is now also used in a directive. Let's call it something that's easy to grep, how about `vocabularySearchString` or something similar